apps/wallet/src/features/Send/InputTransaction/index.tsx

```diff
@@ -30,6 +33,16 @@ const InputTransaction: FC<Props> = ({ navigator }) => {
 		}
 	};
 
+	useEffect(() => {
+		if (collectible) {
+			txActions.update({ type: 'Collectible' });
+			const collection = collections.find(
+				(ele) => ele._id === collectible.collectionId,
+			);
+			txActions.update({ collection });
+		}
+	}, []);
```

Comment:
should include collectible to the dependencies, each time the collectible change -> auto update correspond collection